cime_config/machines/cmake_macros/nvidiagpu.cmake

```diff
@@ -4,14 +4,15 @@ set(MPIFC "ftn")
 set(SCC "nvcc")
 set(SCXX "nvc++")
 set(SFC "nvfortran")
+
 string(APPEND CPPDEFS " -DFORTRANUNDERSCORE -DNO_R16 -DCPRNVIDIA")
 string(APPEND CPPDEFS_DEBUG " -DYAKL_DEBUG")
 if (compile_threaded)
   string(APPEND CMAKE_C_FLAGS " -mp")
 endif()
 string(APPEND CMAKE_C_FLAGS_RELEASE " -O2")
 string(APPEND CMAKE_C_FLAGS_DEBUG " -g")
-string(APPEND CMAKE_Fortran_FLAGS " -i4 -Mstack_arrays -Mextend -byteswapio -Mflushz -Kieee -DHAVE_IEEE_ARITHMETIC -Mallocatable=03 -DNO_R16 -traceback")
+string(APPEND CMAKE_Fortran_FLAGS " -i4 -Mstack_arrays -Mextend -byteswapio -Mflushz -Kieee -DHAVE_IEEE_ARITHMETIC -Mallocatable=03 -DNO_R16 -traceback -acc")
```

Comment:
According to @ndkeen's comment, this setting might be placed in the Depends file, not in this cmake file, in order to limit the scope of impact to specified MPAS files.